src/components/prompts/PromptListRow.tsx

```diff
@@ -13,9 +14,32 @@ interface PromptListRowProps {
   onClickEdit?: (e: React.MouseEvent) => void
   isCta?: boolean
 }
+
 const PromptListRow = ({prompt, icon, type, isCta, onClick, onClickEdit}: PromptListRowProps) => {
+  const handleClick = (e: React.MouseEvent) => {
+    trackEvent('HL Chat Prompt Selected From List', {
+      promptName: prompt.name,
+      promptSlug: prompt.slug,
+      promptType: type,
+      isCta: isCta
+    });
+    onClick(e);
+  };
+
+  const handleEditClick = (e: React.MouseEvent) => {
+    e.stopPropagation();
+    trackEvent('HL Chat Prompt Edit Initiated From List', {
+      promptName: prompt.name,
+      promptSlug: prompt.slug,
+      promptType: type
+    });
```

Comment:
Do these kind of dupe events from above?

Reply:
Removed